build.gradle

```diff
@@ -33,6 +33,24 @@ ext {
     supportLibraryVersion = "24.2.1"
     butterKnifeVersion = "7.0.1"
     daggerVersion = "2.0.2"
+
+    // POM file
+    GROUP = "com.github.brianPlummer"
+    VERSION_NAME = "1.0.0-SNAPSHOT"
```

Comment:
- `-SNAPSHOT` means it will but published to snapshots
- Removing this will publish to mavenCentral